tox.ini

```diff
@@ -24,7 +25,8 @@ description =
     run tests with pytest
 deps =
     dev: :NUMPY_NIGHTLY:numpy
-    dev: git+https://github.com/astropy/astropy#egg=astropy
+    dev: pyerfa
```

Comment:
@Cadair - if you have ideas on avoiding this in future let me know :)

Reply:
Why did this error?

Reply:
Complained PyERFA coudn't be found in the azure feed